CONTRIBUTING.md

```diff
@@ -5,134 +5,51 @@
 
-
-## I don't want to read this whole thing I just have a question!!!
+## I don't want to read this whole thing I just have a question
 
 > **Note:** Please don't file an issue to ask a question. You'll get faster results by reaching out to the community on our [Discord channel](https://cloudquery.io/discord)
 
-
 ## What To Know Before Getting Started
 
 ### CloudQuery Architecture
 
-CloudQuery has a pluggable architecture and is using grpc (go-grpc) to communicate between source plugins, cli and destination plugins. To develop a new plugin for CloudQuery, you don’t need to understand the inner workings of grpc as those are abstracted away via the [plugin-sdk](#cloudquery-plugin-sdk-repo).
-
-CloudQuery has a pluggable architecture and is using the go-plugin to load, run, and communicate between plugins via gRPC. T
-
-![architecture overview](https://docs.cloudquery.io/assets/images/cloudquery-architecture-02b1d1162883cd02510db2cb75c29d27.png)
+CloudQuery has a pluggable architecture and is using [gRPC](https://grpc.io/) to communicate between source plugins, CLI and destination plugins. To develop a new plugin for CloudQuery, you don’t need to understand the inner workings of gRPC as those are abstracted away via the [plugin-sdk](#cloudquery-plugin-sdk-repository).
 
 ### Breakdown of Responsibilities and Repositories
 
-#### CQ CLI and Plugins [Mono Repo](https://github.com/cloudquery/cloudquery):
-- Main entry point and CLI for the user
-- Reading CloudQuery configuration
-- Downloading, verifying, and running plugins
-- Running policy packs
-- Repository for CQ Core can be found 
+#### CloudQuery CLI and Plugins [Mono Repository](https://github.com/cloudquery/cloudquery)
 
-#### CloudQuery Plugin SDK [Repo](https://github.com/cloudquery/plugin-sdk):
-- Interacting with CQ Core for initialization and configuration
-- Helper functions for defining table schemas
-- Methods for testing the resource
-- Framework for running and building a plugin locally
+* Main entry point and CLI for the user
+* Reading CloudQuery configuration
+* Downloading, verifying, and running plugins
+* Running policy packs
 
-#### CloudQuery Plugins and CLI [Mono Repo](https://github.com/cloudquery/cloudquery):
-- Executed only by CQ-Core via gRPC 
-- Interaction with remote data sources:
-    - Initialization of clients
-    - Authentication
-    - Fetching of configuration information
-- More information about developing your own plugin can be found [here](https://docs.cloudquery.io/docs/developers/developing-new-provider)
+#### CloudQuery Plugin SDK [Repository](https://github.com/cloudquery/plugin-sdk)
 
+* Interacting with CloudQuery CLI for initialization and configuration
+* Helper functions for defining table schemas
+* Methods for testing the resource
+* Framework for running and building a plugin locally
 
 ## How Can I Contribute?
 
-### Reporting Bugs
```

Comment:
This seems like a duplication of information that should be in our issue templates